pkg/scanner/local/scan.go

```diff
@@ -309,16 +309,12 @@ func toDetectedMisconfiguration(res ftypes.MisconfResult, defaultSeverity dbType
 	}
 
 	var primaryURL string
-	if strings.HasPrefix(res.Namespace, "builtin.") {
+
+	// empty namespace implies a go rule from defsec, "builtin" refers to a built-in rego rule
+	// this ensures we don't generate bad links for custom policies
+	if res.Namespace == "" || res.Namespace == "builtin" {
```

Comment:
Does the exact match work? i thought namespace could be like `builtin.kubernetes.id200`.

```
res.Namespace == "builtin"
```